### PR TITLE
XDR entity investigation

### PR DESCRIPTION
**Pull request by ArikDay**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4924)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Cortex_XDR_-_Entity_Investigation

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics_README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+This playbook is part of the ‘Cortex XDR by Palo Alto Networks’ pack. This playbook fetch alerts related to specific entity  from Cortex XDR , on a given timeframe, based on MITRE Tactics.
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics_README.md:1]: lets change the "fetch" to other word, can be confusing for customer with Fetching Incidents (integration param)

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics_README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+| EntityType | Entity type to search on xdr-get-alers custom filters |  | Optional |
+| entityID | Entity value |  | Optional |
+
+## Playbook Outputs
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| NextPhaseAlerts | Array size | unknown |
+| PaloAltoNetworksXDR.Alert | Fetched alerts | unknown |
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics_README.md:47]: need to change here as well (also in the playbook yml

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics.yml [hunk 3]
```diff
@@ -0,0 +1,1971 @@
...
+  description: Set to True to hunt for identified alerts with MITRE Impact techniques.
+  playbookInputQuery:
+- key: HuntCredentialAccessTechniques
+  value: {}
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Credential Access techniques.
+  playbookInputQuery:
+- key: timeRange
+  value:
+    simple: 2 hours
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics.yml:1944]: shouldn't it be "2 hours **ago**"?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics.yml [hunk 4]
```diff
@@ -0,0 +1,1971 @@
...
+              left:
+                value:
+                  simple: PaloAltoNetworksXDR.Alert.severity
+                iscontext: true
+              right:
+                value:
+                  simple: critical
+              ignorecase: true
+      contextKey:
+        simple: NextPhaseAlerts
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics.yml:1703]: not necessary next phase :)

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics.yml (hunk 3 above)
Comment (ArikDay) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics.yml:1944]: so it said "ago" on the description but the "time frame" input in "xdr-get-alerts" asks without "ago" so should i will just fix the description

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**